source/style-guide/style/callouts.txt

```diff
@@ -6,9 +6,8 @@ Callouts and Admonitions
 
 .. default-domain:: mongodb
 
-Callouts or admonitions emphasize important or
-helpful information. Use callouts sparingly to avoid clutter on
-the page and to preserve visual impact. 
+Callouts or admonitions emphasize important or helpful information. Use
+callouts sparingly to avoid clutter on the page and to preserve visual impact. 
```

Comment:
Consider rephrasing the second sentence in the form to (x), (y):

```suggestion
Callouts or admonitions emphasize important or helpful information. To
avoid clutter and to preserve visual impact, use callouts sparingly. 
```

Reply:
@JuliaMongo, did you want to incorporate this feedback?

Reply:
Yes, I just forgot to save. Addressed.